src/index.js

```diff
@@ -35,6 +35,14 @@ client.getCommand = function(name) {
 
 require("./commands")(client);
 
+function suggestCommands(cmd, cmdList) {
+  var suggestion = didYouMean(cmd, cmdList);
```

Comment:
The `suggestCommands()` method logs those warnings as per now and returning the suggestion doesn't make sense right :thinking:

Reply:
All those `logger.error()` statements took in 3 arguments and what was the intention behind concatenating the one within `suggestCommands()`.

Reply:
@bkendall Thoughts?

Reply:
Returning the suggestion definitely makes sense. If a suggestion was made, that value can be returned simply as a truthy value. Then, any `if` statement elsewhere can use  the fact that that returns a value to do other logic.

As for concatenating: mostly consistency and ease of reading.